vllm/config.py

```diff
@@ -407,6 +407,8 @@ class ModelConfig:
     available.\n
     - "vllm" will use the vLLM model implementation.\n
     - "transformers" will use the Transformers model implementation."""
+    override_attention_dtype: str = "fp8"
```

Comment:
This should be `None` by default:
```suggestion
    override_attention_dtype: Optional[str] = None
```